### PR TITLE
Update README and versions for 20.03 release

### PR DESCRIPTION
**Pull request by dzier**
Don't push until release

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -27,16 +27,285 @@
 
 |License|
 
-NVIDIA TensorRT Inference Server
-================================
+NVIDIA Triton Inference Server
+==============================
 
-    **NOTE: You are currently on the r20.03 branch which tracks
```
Comment (deadeyegoodwin) [inline README.rst]: Include note about the new name.

Comment (dzier) [reply README.rst]: You want a note at the top here?

Comment (deadeyegoodwin) [reply README.rst]: Look at the master README, there is bold text that starts with "NEW NAME", we want that included in this README as well.

Comment (dzier) [reply README.rst]: Hold, let me fix that then

**Pull request merged**